webui/react/src/pages/FlatRuns/FlatRuns.tsx

```diff
@@ -160,6 +162,7 @@ const FlatRuns: React.FC<Props> = ({ projectId, searchId }) => {
   const sortString = useMemo(() => makeSortString(sorts.filter(validSort.is)), [sorts]);
   const loadableFormset = useObservable(formStore.formset);
   const filtersString = useObservable(formStore.asJsonString);
+  const filterFormSetWithoutId = useObservable(formStore.filterFormSetWithoutId);
```

Comment:
question: does this need to passed down from here? Looks like it's only used in `RunFilterInterstitialModalComponent` so I'm wondering if that component can just import there.

Reply:
if we export `formStore`, i think this doesnt need to be passed, but passing it down makes writing test easier

Reply:
if testing is the only reason to pass it down and it works fine without it, I'd prefer not passing it down and instead mocking the import in tests as necessary.

Reply:
fixed